run.sh

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env sh
+# Convenience script to run the wikiparser with the maps generator as a drop-in replacement for the scraper.
+#
+# 1. Finds latest map build.
+# 2. Builds wikiparser.
+# 3. Extracts wikidata ids and wikipedia urls from generator intermediate files `id_to_wikidata.csv` and `wiki_urls.txt`
+# 4. Runs wikiparser for all input dump files.
+#
+# For more information on the map generator, see
+# <https://github.com/organicmaps/organicmaps/blob/b52b42bd746fdb8daf05cc048f0b22654cfb9b8e/tools/python/maps_generator/README.md>.
+
+USAGE="./run.sh DUMP_FILE.json.tar.gz [DUMP_FILE.json.tar.gz...]
+
+Environment Variables:
+- MAPS_BUILD_ROOT: The directory with all maps builds, defaults to $HOME/maps_build, ignored if MAPS_DIR is set.
+- MAPS_DIR: The map build directory to use for generating descriptions, defaults to the latest directory in MAPS_BUILD_ROOT."
+
+set -eu
+# set -x
+
+# Write printf-style message to stderr with a timestamp and line ending.
+log () {
+    printf "%s " "$(date '+%Y-%m-%dT%H:%M:%SZ')" >&2
+    # shellcheck disable=2059
+    printf "$@" >&2 # Forward all arguments to printf.
+    printf "\n" >&2
+}
+
+if [ -z "${1}" ]
+then
+    printf "Usage:\t%s\n" "$USAGE" >&2
+    exit 1
+fi
+
+# Ensure we're running in the directory of this script.
+SCRIPT_PATH=$(dirname "$0")
+cd "$SCRIPT_PATH"
+SCRIPT_PATH=$(pwd)
+
+# Get latest maps build folder
+
+if [ -z "${MAPS_DIR+}" ]
+then
```

Comment:
nit: here and in other places
```suggestion
if [ -z "${MAPS_DIR+}" ]; then
```